client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/PatientAddress.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Grid, TextField } from '@material-ui/core';
+
+import DBAddress from 'models/DBAddress';
+import StoreStateType from 'redux/storeStateType';
+import FormRowWithInput from 'commons/FormRowWithInput/FormRowWithInput';
+import City from 'models/City';
+
+const ADDRESS_LABEL = 'כתובת';
+const CITY_LABEL = 'עיר';
+const STREET_LABEL = 'רחוב';
+const FLOOR_LABEL = 'קומה';
+const HOUSE_NUM_LABEL = 'מספר בית';
+
+const PatientAddress: React.FC<PatientAddressProps> = (): JSX.Element => {
+
+    const patientAddress = useSelector<StoreStateType, DBAddress>(state => state.address);
+    const cities = useSelector<StoreStateType, Map<string, City>>(state => state.cities);
+
+    const { city, floor, houseNum, streetName: street } = patientAddress;
+
+    return (
+        <FormRowWithInput labelLength={2} fieldName={ADDRESS_LABEL}>
+            <>
+                { city && 
+                    <Grid item xs={2}>
+                        <TextField
+                            disabled 
+                            value={cities.get(city)?.displayName}
+                            label={CITY_LABEL}
+                        />
+                    </Grid> 
+                }
+                { street && 
+                    <Grid item xs={2}>
+                        <TextField
+                            disabled 
+                            value={street}
+                            label={STREET_LABEL}
+                        />
+                    </Grid>
+                }
+                { floor && 
```

Comment:
I think it is better to display it, even if the floor and house num are empty
 maybe we can add לא ידוע or something like this